src/data.js

```diff
@@ -13407,6 +13407,31 @@ module.exports = [
       'Jekyll',
     ],
   },
+  {
+    name: 'Nick Ali',
+    description:
+      'Marketer who used to be a software architect.',
+    url: 'https://nali.org/uses',
```

Comment:
This url has to exist to be merged. Right now it's redirecting to a 404 page.

Reply:
My bad, it's fixed now.